release-notes.adoc

```diff
@@ -1,22 +1,19 @@
 [[rhmap-3.10.0-release-notes-for-wfm]]
-RHMAP 3.10.0 Release notes for WFM
-----------------------------------
+= RHMAP 3.10.0 Release notes for WFM
 
 [[known-issues]]
-Known issues
-~~~~~~~~~~~~
+== Known issues
+
 
 [[cloud-app]]
-Cloud app
-^^^^^^^^^
+=== Cloud app
 
 * After creating an WFM project from the project template, the appform
 ids in the the `lib/app/workflow.js` folder must be updated with new
 appform ids.
 
 [[portal-client]]
-Portal client
-^^^^^^^^^^^^^
+=== Portal client
```

Comment:
The indentation appears to have been stripped from this list during the AsciiDoc migration—all of the points are now at the same level. It looked like this before:

* WFM-portal works in Chrome only
  (Bug #5388)
* Scheduler
  * Unscheduled workorder list is depopulated on selected date change
    (Bug #6260)
  * Only the left-hand-side of a scheduled workorder is draggable
    (Bug #5910)
* Workers
  * Cancel button doesn't work when editing workers
    (Bug #6111)
  * Editing a worker after deleting a group he is in is not possible
    (Bug #6126)